### PR TITLE
Some post-merge fixes

### PR DESCRIPTION
**Pull request by IRacle1**
- Deleted `Visuals939` effect, because deleted in base game
- Fixed `MirrorExtensions::MessageTranslated` method
- Really last fix for phantom pickups (lets say thanks to base game system: the grenade gameObject is destroyed before explosion)

### CODE REVIEW

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 1]
```diff
@@ -212,16 +212,20 @@ public static void PlayCassieAnnouncement(this Player player, string words, bool
...
                 annoucement.Append($"{translations[i]}<size=0> {cassies[i].Replace(' ', ' ')} </size><split>");
 
+            string message = annoucement.ToString();
+
             foreach (RespawnEffectsController controller in RespawnEffectsController.AllControllers)
             {
                 if (controller != null)
                 {
-                    SendFakeTargetRpc(player, controller.netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), annoucement, makeHold, makeNoise, isSubtitles);
+                    SendFakeTargetRpc(player, controller.netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), message, makeHold, makeNoise, isSubtitles);
```
Comment (iopietro) [inline Exiled.API/Extensions/MirrorExtensions.cs:231]:
```suggestion
                    SendFakeTargetRpc(player, controller.netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), StringBuilderPool.Shared.ToStringReturn(announcement), makeHold, makeNoise, isSubtitles);
```
(Remove `StringBuilderPool.Shared.Return(annoucement);` below)

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 2]
```diff
@@ -212,16 +212,20 @@ public static void PlayCassieAnnouncement(this Player player, string words, bool
         public static void MessageTranslated(this Player player, string words, string translation, bool makeHold = false, bool makeNoise = true, bool isSubtitles = true)
         {
             StringBuilder annoucement = StringBuilderPool.Shared.Rent();
```
Comment (iopietro) [inline Exiled.API/Extensions/MirrorExtensions.cs:217]:
```suggestion
            StringBuilder announcement = StringBuilderPool.Shared.Rent();
```

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 3]
```diff
@@ -212,16 +212,20 @@ public static void PlayCassieAnnouncement(this Player player, string words, bool
...
+
             string[] cassies = words.Split('\n');
             string[] translations = translation.Split('\n');
+
             for (int i = 0; i < cassies.Length; i++)
                 annoucement.Append($"{translations[i]}<size=0> {cassies[i].Replace(' ', ' ')} </size><split>");
 
+            string message = annoucement.ToString();
```
Comment (iopietro) [inline Exiled.API/Extensions/MirrorExtensions.cs:225]:
```suggestion
```

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 4]
```diff
@@ -212,16 +212,20 @@ public static void PlayCassieAnnouncement(this Player player, string words, bool
         public static void MessageTranslated(this Player player, string words, string translation, bool makeHold = false, bool makeNoise = true, bool isSubtitles = true)
         {
             StringBuilder annoucement = StringBuilderPool.Shared.Rent();
+
             string[] cassies = words.Split('\n');
             string[] translations = translation.Split('\n');
+
             for (int i = 0; i < cassies.Length; i++)
                 annoucement.Append($"{translations[i]}<size=0> {cassies[i].Replace(' ', ' ')} </size><split>");
```
Comment (iopietro) [inline Exiled.API/Extensions/MirrorExtensions.cs:223]:
```suggestion
                announcement.Append($"{translations[i]}<size=0> {cassies[i].Replace(' ', ' ')} </size><split>");
```

**Review by Thundermaker300 — APPROVED** [review]

**Review by iopietro — CHANGES_REQUESTED** [review]

File: Exiled.API/Extensions/MirrorExtensions.cs (hunk 1 above)
Comment (IRacle1) [reply Exiled.API/Extensions/MirrorExtensions.cs:231]: that code is executed inside `foreach`, so it is impossible to do like that, but i call `StringBuilderPool.Shared.ToStringReturn(announcement)` before to init `message` var

File: Exiled.API/Extensions/MirrorExtensions.cs (hunk 3 above)
Comment (IRacle1) [reply Exiled.API/Extensions/MirrorExtensions.cs:225]: see above

**Review by iopietro — APPROVED** [review]

**Comment by IRacle1** [comment]
dont merge, i found another pickup error...

**Comment by IRacle1** [comment]
💀 now ready

**Review by iopietro — APPROVED** [review]

**Pull request merged**